CRM/Geocoder/Upgrader.php

```diff
@@ -29,6 +29,18 @@ public function upgrade_1100() {
     return TRUE;
   }
 
+	/**
+	 *  Update the URL for Open Street Map end point
+	 *
+	 * @throws \CiviCRM_API3_Exception
+	 */
+	public function upgrade_1200() {
+		$this->ctx->log->info( 'Applying update 1200: Update the URL for Open Street Map end point' );
```

Comment:
For the whole section: No spaces after opening parentheses or before closing parentheses. See [PSR-2](https://www.php-fig.org/psr/psr-2/), section 4.3.

Reply:
@sebalis you are correct re the whitespace - although CiviCRM follows drupal coding standards rather than PSR-2 (sadly we adopted them shorting before PSR-2 became the new VHS). That's the 2 whitespaces rather that 4 gig

Reply:
Doesn't really matter TBH

Reply:
When I made my two comments I had overlooked that the PR had already been merged. My intention was to suggest making these two changes before merging. Discussing this after merging does seem a bit pointless, I agree (hesitantly). I’m sorry for apparently missing the time window to contribute in a more meaningful way. There’s too much else going on for me so all I managed to do is try and catch up with developments very late at night.